3p/iframe-messaging-client.js

```diff
@@ -25,6 +25,8 @@ export class IframeMessagingClient {
   constructor(win) {
     /** @private {!Window} */
     this.win_ = win;
+    /** @private {?string} */
+    this.rtvVersion_ = (win.AMP_CONFIG && win.AMP_CONFIG.v) || null;
```

Comment:
The ad tag is injecting the script (this will be public record for ad tags to integrate amp-inabox).  So to be clear we should expect the version to pass /^[0-9]{15,20}$/?